examples/cpp/SYCLDemo.cpp

```diff
@@ -24,9 +24,9 @@
 // IN THE SOFTWARE.
 // ----------------------------------------------------------------------------
 
-#include "open3d/core/TensorFunction.h"
+#include "open3d/core/SYCLUtils.h"
 
 int main() {
-    open3d::core::SYCLDemo();
+    open3d::core::sycl_utils::SYCLDemo();
```

Comment:
It seems this example is more of a test than an example. We may remove this if `test` is covering this. 
Since Tensor will cover the implementation, an SYCL example on its own might not make much sense. 

We may add `sycl` device in core::Tensor documentation/python example showing tensor copy between CPU, CUDA and SYCL.

Reply:
We need the example since the cmake is different between example and test. This is used to ensure that the SYCL build system is working as expected. The pybind is the same idea.

Once we have actual SYCL implementation, e.g. SYCL Tensor support, then we can remove this example and replace it with the SYCL tensor example.